public/director-upload-movie.php

```diff
@@ -1,15 +1,76 @@
 <?php
-include('../utils/authentication.php');
+// include('../utils/authentication.php');
 
-checkLoggedIn();
+// checkLoggedIn();
 ?>
 <!DOCTYPE html>
 <html lang="en">
 
   <head>
     <title>template</title>
     <link rel="stylesheet" href="./assets/css/styleupload.css">
-    <?php include "../templates/head.php" ?>
+    <?php include "../templates/head.php";
+          require("../utils/movies.php");
+          require("../utils/filter.php");
+          DEFINE('DS', DIRECTORY_SEPARATOR); 
+          
+          $userId = 2;
+
+          if(isset($_POST['upload'])){
+            if(!empty($_POST['MovieTitle']) AND !empty($_POST['Category']) AND !empty($_POST['AgeRating']) AND !empty($_POST['filmGuide'])  AND  !($_FILES['Movie']['error'] > 0) AND !($_FILES['Thumbnail']['error'] > 0) AND !empty($_POST['Description'])){  
+              $title = $_POST['MovieTitle'];
+              $genre = $_POST['Category'];
+              $ageRating = $_POST['AgeRating'];
+              $filmGuide = $_POST['filmGuide'];
+              $description = $_POST['Description'];
+              $moviefile = $_FILES['Movie'];
+              $thumbnailfile = $_FILES['Thumbnail'];
+              $allowed = array("mp4","mp3");
```

Comment:
Regel 28, zou je ipv een nieuwe array te kunnen maken, de variabele $allowed gelijk kunnen zetten aan de waarde van de constante variabele in /src/constants ?

Dit is voor regel 28 zo voor de video extensies en regel 44 voor de image extensies